routemaster/jobs/fail.rb

```diff
@@ -0,0 +1,15 @@
+require 'routemaster/mixins/log'
+
+module Routemaster
+  module Jobs
+    class Fail
+      include Mixins::Log
+
+      def call(*args)
+        raise 'failing job'
+      end
+    end
+  end
+end
+
```

Comment:
No good reason as the job list is short. And startup time wouldn't be changed much!
Thanks for prodding, I'll tweak this.